README.md

```diff
@@ -74,5 +74,3 @@ Pull requests are appreciated. Please use the same [Contributor License Agreemen
 ---
 
 Developed by the Cesium team.
-
```

Comment:
Please change all URLs in this file from `AnalyticalGraphicsInc` -> `CesiumGS`, and `cesiumjs.org` -> `cesium.com`, or use best judgement as needed.

Reply:
Nice catch. You may have noticed that the `README.MD` file in the branch was simply a relic from the last webpack tutorial that we released. It is mostly relevant to people who view our GitHub page, not users accessing our tutorial. I just added a commit that addresses this issue.